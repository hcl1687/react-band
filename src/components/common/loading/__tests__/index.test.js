import { render, shallow } from 'enzyme'
import React from 'react'
import loadingFactory from '../index.entry'

const Loading = loadingFactory()

describe('common/loading', () => {
  it('should render correctly', () => {
    const __ = key => key
    const wrapper = render(<Loading __={__} />)
    expect(wrapper).toMatchSnapshot()
  })

  it('should invoke i18n translator', () => {
    const __ = jest.fn()
    const wrapper = shallow(
      <Loading __={__} />
    )
    expect(wrapper.find('div').length).toBe(1)
    expect(__).toHaveBeenCalled()
  })
})
