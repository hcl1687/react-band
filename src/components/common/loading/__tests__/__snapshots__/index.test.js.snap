// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/loading should render correctly 1`] = `
<div>
  loading
</div>
`;
